src/collar/oc_com.lsl

```diff
@@ -566,42 +681,60 @@ default {
         if (g_lWrongRootScripts) {
             sMessage = "\nFalse root prim placement:\n";
             do {
-                sMessage += llList2String(g_lWrongRootScripts,0);
-                g_lWrongRootScripts =  llDeleteSubList(g_lWrongRootScripts,0,0);
+                sMessage += llList2String(g_lWrongRootScripts, 0);
+                g_lWrongRootScripts = llDeleteSubList(g_lWrongRootScripts, 0, 0);
             } while (g_lWrongRootScripts);
         }
-        if(sMessage) sMessage += "\n";
+
+        if (sMessage) {
+            sMessage += "\n";
+        }
+
         integer i;
-        integer index;
+        integer iIndex;
         list lTemp = ["Missing Scripts:"];
         do {
-            index = llListFindList(g_lFoundCore5Scripts,llList2List(g_lCore5Scripts,i,i));
-            if (index == -1) {
-                if (llSubStringIndex(sMessage,llList2String(g_lCore5Scripts,i+1)) == -1)
-                    lTemp += [llList2String(g_lCore5Scripts,i+1)];
-            } else
-                sMessage += "\n"+llList2String(g_lCore5Scripts,i+1) + "\t(Link# "+llList2String(g_lFoundCore5Scripts,index+1)+")";
-            i = i + 2;
-        } while (i<10);
+            iIndex = llListFindList(g_lFoundCore5Scripts, llList2List(g_lCore5Scripts, i, i));
+            if (iIndex == -1) {
+                if (llSubStringIndex(sMessage, llList2String(g_lCore5Scripts, i + 1)) == -1) {
+                    lTemp += llList2String(g_lCore5Scripts, i + 1);
+                }
+            } else {
+                sMessage += "\n" + llList2String(g_lCore5Scripts, i + 1) + "\t(Link# " + llList2String(g_lFoundCore5Scripts, iIndex + 1) + ")";
+            }
+
+            i += 2;
+        } while (i < 10);
+
         i = llGetLinkNumber();
-        if (i != 1) sMessage += "\noc_com\t(not in root prim!)";
+        if (i != 1) {
```

Comment:
Somehow i feel like this is cleaner... but at the same time, its going to make a lot of scripts a lot longer.

Reply:
Why not, if it makes code well-readable? I suppose the main target is to write good code, not minified.